src/watchkit.cs

```diff
@@ -1576,6 +1597,20 @@ enum WKInterfaceVolumeControlOrigin : long {
 		Companion,
 	}
 
+	[Watch (6,1), NoiOS]
+	[Native]
+	enum WKInterfaceAuthorizationAppleIDButtonStyle: long {
+		Default,
+		White,
+	}
+
+	[Watch (6,1), NoiOS]
```

Comment:
Is the `NoiOS` necessary?

Reply:
Yes, the parent interface has properties/methods that also support iOS.

Reply:
I don't think it's needed (but there's no harm in it).
WatchKit used to be available on iOS - but it's not anymore (and there are stubs to maintain binary compatibility).